package.json

```diff
@@ -29,5 +29,8 @@
     "release-it-yarn-workspaces": "^2.0.0",
     "typescript": "^4.7.4"
   },
+  "resolutions": {
+    "ember-cli-htmlbars": "^6.0.1"
+  },
```

Comment:
@dfreeman Without this, TS can end up mis-resolving it to the v5.x present transitively in the dependency graph when doing the `integration-declarations.d.ts` module merge for `ember-cli-htmlbars`. (That would seem to be a bug in TS' resolutions, given that we *explicitly* ask for v6!) I discovered this after beating my head against the wall for over an hour trying to figure out why all the *other* declaration merges in the integration declarations were working, but that one was not—and in fact was simply not being included at all. In the end, `traceResolution` told me that it *was* being merged… with the wrong version of `ember-cli-htmlbars`'s types. 🙃

I have some concerns about using `resolutions` to fix this long-term: I *think* this will ensure that all of our own packages will work correctly this way, but I want to get it cleaned up *ASAP* by updating the things we use which transitively depend on the "wrong" version. I'm also somewhat horrified that TS can mis-resolve it that way, and if I can come up with a minimal reproduction of it, I will file a bug on TS.

Reply:
I could also imagine this being a "Yarn is imprecise about monorepo management" issue if v5 happened to be the one to get hoisted and v6 was only present for the specific app(s) that requested it, since packages from the monorepo will always see the top-level version of a dependency unless they themselves have requested something else.

This might be a good reason to consider shifting to npm or pnpm here (both of which are stricter about getting `peerDependencies` right), though I do wish Volta and Dependabot had proper support for the latter.

Reply:
Thankfully pnpm support is in flight; I would be very happy to switch as soon as it lands.

Reply:
Err: Volta had pnpm support in flight. Dependabot is not getting a ton of attention since being acquired AFAICT; we should reevaluate Renovate if that ends up being a blocker.